internal/eval/partial_test.go

```diff
@@ -1212,7 +1212,7 @@ func TestPartialBasic(t *testing.T) {
 func TestPartialPanic(t *testing.T) {
 	t.Parallel()
 	testutil.Panic(t, func() {
-		partial(Env{}, nil)
+		_, _ = partial(Env{}, nil)
```

Comment:
I'm fine either way, but I think there's a //lint:noerrcheck or something to that effect that is an alternative, here.